packages/client-core/src/common/services/NotificationService.tsx

```diff
@@ -23,13 +23,15 @@ All portions of the code written by the Ethereal Engine team are Copyright © 20
 Ethereal Engine. All Rights Reserved.
 */
 
-import { SnackbarProvider, VariantType } from 'notistack'
+import { SnackbarKey, SnackbarProvider, VariantType, closeSnackbar } from 'notistack'
+import React, { CSSProperties, Fragment, useEffect, useRef } from 'react'
 
 import multiLogger from '@etherealengine/common/src/logger'
 import { AudioEffectPlayer } from '@etherealengine/engine/src/audio/systems/MediaSystem'
-import { defineState, getState } from '@etherealengine/hyperflux'
+import { defineState, getState, useMutableState } from '@etherealengine/hyperflux'
 
-import { defaultAction } from '../components/NotificationActions'
+import Icon from '@etherealengine/ui/src/primitives/mui/Icon'
+import IconButton from '@etherealengine/ui/src/primitives/mui/IconButton'
```

Comment:
we can use `HiXMark` or similar react-icon instead of using mui icon

Reply:
i've just copied the implementation we have, i'd consider this out of scope for now